src/backend/launcher.ts

```diff
@@ -336,7 +331,12 @@ async function prepareWineLaunch(
       LogPrefix.Backend
     )
     if (runner === 'gog') {
-      await setup(appName)
+      await gogSetup(appName)
+      sendFrontendMessage('gameStatusUpdate', {
+        appName,
+        runner: 'gog',
+        status: 'playing'
+      })
```

Comment:
New setup updates status with redist installation, we are making sure it's set to playing here